Assets/Scripts/Models/Furniture.cs

```diff
@@ -404,11 +440,19 @@ public void ReadXmlPrototype(XmlReader readerParent)
             case "Description":
                 reader.Read();
                 description = reader.ReadContentAsString();
-                break;
+                    break;
```

Comment:
Yeah, for some bizarre reason, sometimes when I paste into a line (such as pasting something that starts with a Carriage Return onto the end of a line) it will indent it another level.